database/factories/TaskFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Str;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Task>
+ */
+class TaskFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition()
+    {
+        return [
+            'task_title'=>fake()->name(),
+            'task_content'=>Str::random(100),
```

Comment:
```suggestion
            'task_title' => fake()->name(),
            'task_content' => Str::random(100),
```
sua tuong tu cac cho khac nha